addons/sale/models/sale.py

```diff
@@ -608,7 +608,8 @@ def action_quotation_send(self):
             'mark_so_as_sent': True,
             'custom_layout': "mail.mail_notification_paynow",
             'proforma': self.env.context.get('proforma', False),
-            'force_email': True
+            'force_email': True,
+            'model_description': self.type_name or False
```

Comment:
I don't think type_name can ever be None because of the compute so you can just write `self.type_name`